index.src.html

```diff
@@ -459,6 +465,48 @@
     to guide the discovery process (by choosing a federated identity provider, BTLE device, etc).
   </div>
 
+  ### Mediation Requirements ### {#mediation-requirements}
+
+  When making a request via {{get(options)}}, developers can set a case-by-case requirement for
+  [=user mediation=] by choosing the appropriate {{CredentialMediationRequirement}} enum value.
+
+  Note: The [[#user-mediation]] section gives more detail on the concept in general, and its
+  implications on how the user agent deals with individual requests for a given origin).
+
+  <pre class="idl">
+    enum CredentialMediationRequirement {
+      "silent",
+      "optional",
+      "required"
+    };
+  </pre>
+  <dl dfn-for="CredentialMediationRequirement" dfn-type="enum-value">
+    :   <dfn>silent</dfn>
+    ::  User mediation is suppressed for the given operation. If it can be performed without user
+        involvement, wonderful. If user involvement is necessary, then the operation will fail
+        rather than involving the user.
+
+        Note: The intended usage is to support "Keep me signed-into this site" scenarios, where
+        a developer may wish to silently obtain credentials if a user should be automatically
+        signed in, but to delay bothering the user with a sign-in prompt until they actively
+        choose to sign-in.
+
+    :   <dfn>optional</dfn>
+    ::  If credentials can be handed over without user mediation, they will be. If no such
+        credentials are available, then the user agent will ask the user for permission.
+
```

Comment:
Note: The intended usage is to support a sign-in button, where showing a credential picker is in context and would not surprise the user.